src/components/Pf/components/form/PfForm.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { hyphenate } from '../../../../utils/ComponentUtils';
+import { Form, FormGroup, TextInput } from '@patternfly/react-core';
+
+interface VAlignFormProps {
+    title?: string;
+    identification?: string;
+    showAsRequired?: boolean;
+    showPopover?: boolean;
+    showHelpText?: boolean;
+    showInputField?: boolean;
+    showInputFieldAsRequired?: boolean;
+    popoverMessage?: string;
+    helpText?: string;
+    children?: React.ReactNode;
+}
+
+const formName = 'simple-form';//label name
+const formInputName = 'simple-form-input';//label name
+
+// Uses the following component to create i)Form labels with optional
+// ii)required iii)popover iv)HelpText, etc..
+export class VAlignForm extends React.Component< VAlignFormProps> {
```

Comment:
Just a quick comment, you are using a class component here. 
This is not wrong, but we are using [Function components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) to be able to use [Hooks](https://reactjs.org/docs/hooks-intro.html).